web/client/components/mapcontrols/annotations/AnnotationsEditor.jsx

```diff
@@ -447,6 +451,16 @@ class AnnotationsEditor extends React.Component {
                     onUnselectFeature={this.props.onResetCoordEditor}
                 />
                 }
+                {
+                    isEditMeasure && <Button className="btn btn-primary measure" onClick={()=> {
+                        const editProperties = this.props.editing?.properties;
+                        // Excluding geometry of type Point as they are labels of measurement
+                        const features = editProperties?.type === 'Measure' && this.props.editing.features.filter(f=> f.geometry.type !== 'Point');
```

Comment:
can this type be a constant and replace all the usage of the string "Measure"?
like in utils or so put
export const MEASURE_TYPE = "Measure";